### PR TITLE
Add kinetic model view

### PR DESCRIPTION
**Pull request by kianmehrabani**

### CODE REVIEW

File: database/models/reaction_species.py [hunk 1]
```diff
@@ -188,7 +188,23 @@ def to_rmg(self):
...
+        left_side = " + ".join(
+            f"{int(abs(stoich)) if abs(stoich) != 1 else ''}{species.formula}"
+            for stoich, species in stoich_reactants
+        )
+        right_side = " + ".join(
+            f"{int(stoich) if stoich != 1 else ''}{species.formula}"
+            for stoich, species in stoich_products
+        )
+
+        return f"{left_side} -> {right_side}"
```
Comment (rwest) [inline database/models/reaction_species.py:207]: I can't tell where this is used but at some point and is some contexts we might care about writing reversible reactions as <=> and irreversible ones as ->

File: database/templates/database/kineticmodel_detail.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+{% block content %}
+<h1>{{kinetic_model.model_name}}</h1>
+
+<dl>
+    {% if kinetic_model.chemkin_reactions_file or kinetic_model.chemkin_transport_file or kinetic_model.chemkin_thermo_file %}
+    <dt>Downloads</dt>
+        {% if kinetic_model.chemkin_reactions_file %}
+        <dd><a href="{{ kinetic_model.chemkin_reactions_file.url }}" download>Chemkin Reactions File</a></dd>
+        {% endif %}
+        {% if kinetic_model.chemkin_transport_file %}
```
Comment (rwest) [inline database/templates/database/kineticmodel_detail.html:14]:
```suggestion
        {% if kinetic_model.chemkin_thermo_file %}
```

File: database/templates/database/kineticmodel_detail.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <dd><a href="{{ kinetic_model.chemkin_thermo_file.url }}" download>Chemkin Thermo File</a></dd>
+        {% endif %}
+        {% if kinetic_model.chemkin_thermo_file %}
```
Comment (rwest) [inline database/templates/database/kineticmodel_detail.html:17]:
```suggestion
        {% if kinetic_model.chemkin_transport_file %}
```

File: database/templates/database/kineticmodel_detail.html (hunk 2 above)
Comment (kianmehrabani) [reply database/templates/database/kineticmodel_detail.html]: 2838e09

File: database/models/reaction_species.py (hunk 1 above)
Comment (kianmehrabani) [reply database/models/reaction_species.py]: 4c824da

**Pull request merged**